hnn_core/dipole.py

```diff
@@ -256,7 +256,15 @@ class Dipole(object):
 
     def __init__(self, times, data, nave=1):  # noqa: D102
         self.times = np.array(times)
-        self.data = {'agg': data[:, 0], 'L2': data[:, 1], 'L5': data[:, 2]}
+
+        if data.ndim == 1:
+            data = data[:, None]
```

Comment:
What's this for?

Reply:
If you provide a 1D array, then data.shape[1] doesn't exist down below. For e.g., data.shape = (6000,) will become (6000, 1)

Reply:
I tried using `np.atleast_2d` first but then it made it (1, 6000) so I just went with my manual solution

Reply:
I see, that makes sense!